scripts/copy_declarations.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+TEMP_DIR=$1
+
+echo "copying declaration.d.ts to types/common/declaration.d.ts"
```

Comment:
Why do we need this script? Can we provide docworks with the dir we want from the start?
i.e, change:
https://github.com/wix-incubator/corvid-types/blob/7972d8e4f7414216d13b6f2198b05375814b906e/package.json#L9
to `-d types/common`